Hannah-Sten/TeXiFy-IDEA
# Improved handling of bibinputs environment variable

Fix #1814

#### Summary of additions and changes

* Add inspection to warn against bug in bibtex which doesn't always handle relative paths correctly
* Take bibinputs into account when resolving to a bib file
* Move test files for inspections to a similar folder structure as source files

#### How to test this pull request

* Take your favourite bibtex mwe, then move the bibliography one directory higher and use `\bibliography{../mybib}`, create run configuration from context using pdflatex so it will use bibinputs.
* Note that it doesn't compile.
* Use quickfix for the inspection that complains about the relative path
* Now it compiles, and the reference is resolved correctly


#### Wiki
